src/linker/Linker/Driver.cs

```diff
@@ -363,6 +363,9 @@ public void Run (ILogger customLogger = null)
 						case "overrideremoval":
 							context.DisabledOptimizations |= CodeOptimizations.OverrideRemoval;
 							break;
+						case "lazybodymarking":
```

Comment:
this needs better naming with documentation for the command like option

Reply:
I went with "unreachablebodies"  are you OK with that?

Added command line docs